src/client/ns_turn_msg_defs.h

```diff
@@ -60,9 +60,9 @@
 #define GET_STUN_ERR_RESP(msg_type) (msg_type | 0x0110)
 
 /* Lifetimes: */
-#define STUN_DEFAULT_ALLOCATE_LIFETIME (600)
+#define STUN_DEFAULT_ALLOCATE_LIFETIME (20)
```

Comment:
Were these changed for debugging?
Those should be reverted

Reply:
thanks for catching this, those commits weren't supposed to be in this PR. i took them out however i still have other suggestions to consider.